lib/clouds/libcloud_common.py

```diff
@@ -46,162 +46,108 @@ class LibcloudCmds(CommonCloudFunctions) :
-
              This class doesn't actually interpret your parameters, but it does use the string to interpret
              whether or not your configuration file has been setup correctly.
-
              Libcloud credentials must be in this format, and include a "tag" which is an arbitrary string
              that corresponds to the name of the tenant that owns the credentials.
-
              You can have as many credentials (tenants) as you want, but they need to be in the right format.
-
              Example 1)
                 1. Cloud is named "FOO"
                 2. There are two tenants "user1" and "user2"
                 3. Each tenant only has one (1) credential for authentication: "bar" and "baz".
                 4. The configuration file would look like this:
-
                  [USER-DEFINED : CLOUDOPTION_FOO ]
                  FOO_CREDENTIALS = user1:bar,user2:baz
-
              Example 2)
                 1. Cloud is named "HAPPY"
                 2. There are three tenants named "not", "so", and "lucky"
                 3. Each tenant has the same access_key and access_token, two (2) credentials each as "foo" and "bar"
                 4. The configuration file would look like this:
-
                  [USER-DEFINED : CLOUDOPTION_HAPPY ]
                  HAPPY_CREDENTIALS = not:foo:bar,so:foo:bar,lucky:foo:bar
-
             In Example 1), @num_credentials = 1
             In Example 2), @num_credentials = 2
-
             By, default we assume Oauth-driven clouds that have both an access token and an access key.
-
             If you have more or less than that, please set the value accordingly.
-
      @use_ssh_keys: (Optional for cloud-init-based clouds, Default false)
             A comma/colon-separated list of ssh key identifiers that can be looked up by libcloud and installed by cloud-init.
-
             For example:
                 1. Cloud is named "FOO"
                 2. two SSH keys, identified by "a" and "b"
                 3. The configuration file would look like this:
-
                  [USER-DEFINED : CLOUDOPTION_FOO ]
                  FOO_KEY_NAME = a,b # identifiers used by cloud-init to install the public key
                  FOO_SSH_KEY_NAME = path/to/private/key/for/cbtool
-
             Using libcloud, we will then go and lookup those keys and use them at instance creation time.
-
             NOTE: FOO_SSH_KEY_NAME != FOO_SSH_KEY_NAME. The first one is the private key used by cbtool,
                   and the second one is the public key.
-
             NOTE: If your cloud doesn't support cloud-init, then you will need to make sure the VM image is prepared in advance
                   with the public key that corresponds to FOO_SSH_KEY_NAME
-
     @use_cloud_init: (Optional, Default False)
             NOTE: If your cloud doesn't support cloud-init, you will need to make sure your images have baked in the SSH public key on your own.
-
-    @use_volumes: (Optional, Default False)
```

Comment:
for every adapter (including the upper-layer libcloud one), there is now a "create volume" function, which can be either "real" or "fake"... if the user specifies cloud_vv=<something> in a cloud which does not support volumes, then the "fake" version of the function will be called, which will not actually create a real volume, but it will not cause any error...maybe there is another problem that I am missing...